apps/omg_child_chain/lib/omg_child_chain/release_tasks/set_fee_feed_adapter_opts.ex

```diff
@@ -27,53 +31,72 @@ defmodule OMG.ChildChain.ReleaseTasks.SetFeeFeedAdapterOpts do
   def load(config, _args) do
     _ = on_load()
 
-    @app
-    |> Application.get_env(@config_key)
-    |> update_adapter_opts_with_env(config)
+    existing_config = Application.get_env(@app, @config_key)
+
+    @env_fee_adapter
+    |> System.get_env()
+    |> parse_adapter_value()
+    |> case do
+      "FEED" -> configure_feed_adapter(existing_config, config)
+      _ -> config
```

Comment:
seems like in this case we're not margining into `Config.Reader.merge/2`?
is this not tested?

Reply:
In this case no config is changed so I'm passing the original `config` back. Refactored in https://github.com/omisego/elixir-omg/pull/1385/commits/d2b16cad7cc7da18d12d703af6f4a68c64c375db to make this clearer though.

Reply:
And the test for this is this one https://github.com/omisego/elixir-omg/pull/1385/files#diff-123f6adf27c8ab33907e154dfc96b071R103-R110

Reply:
ahhh shit. my bad!